### ISSUE TITLE
wrong util/util and missig add-foundation-colors in settings file

### ISSUE THREAD

**Issue by DanielRuf** [zurb/foundation-sites#11037]
I tried enabling the settings file directly in scss/foundation.scss.

It seems there is at least one mixin missing when generating a new settings.scss file with `gulp deploy:settings`.

```scss
@import 'util/util'; // wrong path in settings.scss, should be ../util/util?
@include add-foundation-colors; // missing in settings.scss
```

**Comment by DanielRuf** [comment]
Related: https://github.com/zurb/foundation-sites/issues/10537

**Comment by ncoden** [comment]
This is an old issue. See https://github.com/zurb/foundation-sites/issues/8293

`settings/_settings.scss` should not be used directly but copied into the project repo.

**Comment by ncoden** [comment]
See https://foundation.zurb.com/sites/docs/sass.html#the-settings-file

**Comment by DanielRuf** [comment]
Sure but then the comment is misleading.

https://github.com/zurb/foundation-sites/blob/4e21634356da296edd2f1c3e65d91140f991008b/scss/foundation.scss#L19-L21

**Comment by ncoden** [comment]
> Sure but then the comment is a bit misleading.

Because that file too can be copied. You should obviously not edit the Foundation source files themselves. But I agree this can be misleading

**Comment by ncoden** [comment]
> The new generated settings file will not include it. (add-foundation-colors)

It does ;) https://github.com/zurb/foundation-sites/blame/develop/scss/settings/_settings.scss

**Comment by DanielRuf** [comment]
> You should obviously not edit the Foundation source files themselves.

Was just for directly testing and to see what breaks if other people just copy the whole dir (easiest step for some).

> It does ;) https://github.com/zurb/foundation-sites/blame/develop/scss/settings/_settings.scss

Not after a new `deploy:settings`, please check that.

**Comment by DanielRuf** [comment]
The last two changes were manual and https://github.com/zurb/foundation-sites/commit/a8ad9688929726d05d8ec3e4f911e6760bdcb3aa#diff-35a517bca933d90868763d8450d1da1e would be reverted by the next execution of `deploy:settings`.

https://github.com/zurb/foundation-sites/commits/develop/scss/settings/_settings.scss

**Comment by ncoden** [comment]
You're right. That's a problem https://github.com/zurb/foundation-sites/blob/7a5225de045671ce84d69cdf7a7e2f73035676ea/gulp/tasks/deploy.js#L98 --'

**Comment by DanielRuf** [comment]
Do we have to do something here or will we test and tackle this before the next release?

**Comment by DanielRuf** [comment]
It still breaks the next settings.scss and CSS generation (on deploy).

We should check and fix this.

**Comment by ncoden** [comment]
We have three solution:
* Update [octophant](https://github.com/zurb/octophant) so it can generate a SCSS file with what we want
* Update [octophant](https://github.com/zurb/octophant) to use a template (maybe handlebars)
* Update the file manually

One thing is certain, there is a lof of confusions around the `_settings.scss` files. Every new contributors think he/she must update the file, and lot of people expect to import the file as is without copying it into their own project.

**Comment by DanielRuf** [comment]
Also https://github.com/zurb/foundation-sites/commit/a8ad9688929726d05d8ec3e4f911e6760bdcb3aa#diff-35a517bca933d90868763d8450d1da1e will be gone then (which should be in custom projects or templates and not Foundation itself).

Option 1 and 2 should be preferred as option 3 can be error prone.

**Comment by ncoden** [comment]
> Also a8ad968#diff-35a517bca933d90868763d8450d1da1e will be gone then

That's not a problem ;)

**Comment by DanielRuf** [comment]
Still a problem as this loads `node_modules/util/util.js`in many cases.

**Comment by joeworkman** [comment]
This issue has been mentioned on **Foundation Open Source Community**. There might be relevant details there:

https://foundation.discourse.group/t/issues-with-ruby-on-rails-and-foundation-sites-6/2356/22

**Comment by DanielRuf** [comment]
We should add a comment which mentions this and point to relevant setups. Not sure if the ZURB template is allso broken.

**Comment by DanielRuf** [comment]
Ah, we solved it there with `includePaths`:

https://github.com/foundation/foundation-zurb-template/blob/master/gulpfile.babel.js#L96

https://github.com/foundation/foundation-zurb-template/blob/master/config.yml#L23
